### DISCUSSION TITLE
How can I apply blur on the borders of the image to make the borders smoother?

### DISCUSSION THREAD

**Discussion by guga1109** [SixLabors/ImageSharp#2448] [Q&A]
When I use the GaussianBlur method on ImageSharp, the blur is only applied inside the image shape. Because of that, the transition between the blurred image and the background isn't smooth.

In the image below, the blurred image looks like it's joined with the background.
<img width="524" alt="Screenshot 2023-05-05 at 09 38 37" src="https://user-images.githubusercontent.com/36571491/236459629-8100148d-5aa8-4451-a76e-771fa5c1a740.png">

Is there any way I can do it with ImageSharp?

**Comment by JimBobSquarePants** [comment]
You can't blur past the edge of a pixel buffer since there's nothing to blur. However, if the background is a single color, you could perform a padded resize using that color as the background and blur the whole thing before overlaying it on your background.
